README.md

```diff
@@ -9,11 +9,49 @@ modules that read chain state directly.
 
 ## Versioning
 
-Blockchain state versioning does not naturally align with common semantic versioning conventions.
+We adopt a policy similar to the [Builtin-Actors repository](https://github.com/filecoin-project/builtin-actors?tab=readme-ov-file#versioning), with a key difference:
 
```

Comment:
```suggestion
We adopt a policy similar to the [Builtin-Actors repository](https://github.com/filecoin-project/builtin-actors?tab=readme-ov-file#versioning), with these key differences:
```

In practice it is one difference, but given we have two bullets, I changed to "differences".